RecoTracker/FinalTrackSelectors/plugins/powN.h

```diff
@@ -0,0 +1,39 @@
+namespace {
```

Comment:
in principle MultiTrackSelector should go away once HLT is migrated to the new selector